xapian-letor/Makefile.am

```diff
@@ -67,9 +67,34 @@ coverage-check:
 	rm -rf lcov
 	genhtml --demangle-cpp --branch-coverage $(GENHTML_ARGS) -q -p "`pwd`" -t "Test Coverage for xapian-letor `git log -n1 --abbrev-commit --format=%h`" -o lcov xapian-letor.lcov
 
+# We want to clean out build tree paths from the copy of xapianletor-config we
+# install, mostly in the interests of facilitating reproducible builds.
+bin_SCRIPTS = installable/xapianletor-config
+
+# Create the installable version in a subdirectory.  It's not made executable
+# until install time, which hopefully should avoid people trying to use it when
+# building against an uninstalled xapian-core.
```

Comment:
The comment should say `xapian-letor` here.

Reply:
fixed in a fixup commit.